lib/manager/bundler/locked-version.ts

```diff
@@ -0,0 +1,24 @@
+const DEP_REGEX = '(?<=\\().*(?=\\))';
+export function extractLockFileEntries(lockFileContent: string): any {
+  const gemLock = { deps: [] };
+  lockFileContent.split('\n').forEach(eachLine => {
+    const whitespace = eachLine.indexOf(eachLine.trim());
+    // as per original ruby lockfile parser,a line whitespace 2,4,6 contains dependencies.
+    if (whitespace === 4) {
```

Comment:
Can be sure it's only 4 in our case?

Reply:
4 is the place where the main dependencies are listed. Anything greater than 4 is sub dependency and we don't need them for now.
I can add a couple of more gemfile in test form popular repositories to verify that.

Reply:
We should also make sure to only extract from relevant sections.

Reply:
Then we have to narrow down the extraction section only from the GEM section inside lock file. 
Are you suggesting the same?

Reply:
Yes. If we only need this section. Otherwise we maybe can get wrong results.